CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 cmake_minimum_required(VERSION 3.13)
 project(ArborX CXX)
 
-find_package(kokkos REQUIRED)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
```

Comment:
```suggestion
list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
```

Reply:
Any opinion on whether to prepend or append?

Reply:
Should not matter, but I would use prepend.

Reply:
I mostly see projects appending, but maybe prepending is not a bad idea if we don't trust others writing a better `FindKokkos.cmake` file. I don't have a strong opinion here.